package.json

```diff
@@ -16,9 +18,13 @@
   ],
   "author": "Marshall Farrier",
   "license": "MIT",
-  "readmeFilename": "README.md",
-  "gitHead": "2c11de9333ca6ba920bd670ad637fdf50f99f323",
-  "devDependencies": {
-    "grunt": "~0.4.5"
+  "files": [
+    "LICENSE",
+    "README.md",
+    "lib/"
+  ],
+  "dependencies": {
+    "chai": "^2.2.0",
+    "mocha": "^2.2.4"
```

Comment:
Honestly, I'm just coming from what happens using `npm install chai --save`. Looking at the docs, I like '~' better. Agreed?